engine/schema/src/main/java/com/cloud/storage/VolumeVO.java

```diff
@@ -48,6 +48,9 @@ public class VolumeVO implements Volume {
     @Column(name = "id")
     long id;
 
+    @Column(name = "previous_id")
```

Comment:
@FelipeM525 thanks for the PR, LGTM. Minor nit - could we call it last_id ? or last_uuid? This is because in the DB schema, there are many `last...` columns.